portfolio/src/main/webapp/index.html

```diff
@@ -18,6 +18,24 @@ <h1>Server Stats</h1>
       <p>Click here to see JSON string:</p>
       <button onclick="getJSONString()">Fetch JSON string!</button>
       <ul id="json-string"></ul>
+
+      <h1>Modify your word</h1>
+      <form action="/data" method="POST">
+
```

Comment:
nit: typically we don't include blank lines in html. It's a bit confusing since they don't show up in the actual page.

Also we typically use CSS for spacing instead of <br/> tags but I'm okay with the break tags for now.